ocs_ci/helpers/dr_helpers.py

```
@@ -1864,10 +1806,19 @@ def get_managed_cluster_node_ips():
-        cluster.append(get_fencing_cidrs_from_drclusterconfig(cluster[0]))
+        logger.info(f"Getting worker node IPs on managed cluster: {cluster[0]}")
+        node_obj = ocp.OCP(kind=constants.NODE).get()
+        external_ips = []
+        for node in node_obj.get("items"):
+            # Check if node has worker role
+            labels = node.get("metadata", {}).get("labels", {})
+            if "node-role.kubernetes.io/worker" in labels:
+                addresses = node.get("status", {}).get("addresses", [])
+                for address in addresses:
+                    if address.get("type") == "ExternalIP":
+                        external_ips.append(address.get("address"))
```

Ruff (0.15.15)

[warning] Use `list.extend` to create a transformed list

(PERF401)